build-logic/basics/src/main/kotlin/build-logic.reproducible-builds.gradle.kts

```diff
@@ -19,6 +19,10 @@ tasks.withType<AbstractArchiveTask>().configureEach {
     // Ensure builds are reproducible
     isPreserveFileTimestamps = false
     isReproducibleFileOrder = true
-    dirMode = "775".toInt(8)
-    fileMode = "664".toInt(8)
```

Comment:
The other places used to have `755 / 644` rather than `775 / 664`, so I went with `755 / 644`